generators/entity/index.js

```diff
@@ -83,13 +82,23 @@ module.exports = EntityGenerator.extend({
             defaults: ''
         });
 
+        // This adds support for a `--skip-server` flag
+        this.option('skip-server', {
+            desc: 'Skip the server-side application generation',
+            type: Boolean,
+            defaults: false
+        });
+
         this.regenerate = this.options['regenerate'];
         this.entityTableName = this.options['table-name'] || this.name;
+        this.entityName = _s.capitalize(this.name);
         this.entityTableName = _s.underscored(this.entityTableName).toLowerCase();
         this.entityAngularJSSuffix = this.options['angular-suffix'];
+        this.skipServer = this.options['skip-server'];
```

Comment:
I think `skipServer` should be decided based on below and not just based on the passed option
1. If the apps `yo-rc` has `skipServer` property that should be used
2. If its a gateway from where entity is being generated and if the `useMicroserviceJson` option is true we should set `skipServer` to true
3.if its a gateway and if `useMicroserviceJson` is not true we should use what is passed in the option `--skip-server`
4. if its a normal app and if apps yo-rc doesnt have `skipserver` set we should use whats passed in `--skip-server` option